### PR TITLE
Add use-cases: DeleteTask, BanUser, UnbanUser

### LINKED ISSUE #29: Add plan and business-workers characteristics
Add plan and business-workers characteristics to `stakeholders-needs.md` with the next structure:
- Plan of the document
_[Розділ містить опис того, про що йдеться в цій частині цього документу, що залишилася. Також тут описана структура документу.]_
- Roles and proccesses characheristics:

_[В цьому розділі визначаються зовнішні фактори, що впливають на бізнес (бізнес-актори), та внутрішні фактори (робітники), дається загальна характеристика діяльності бізнес-акторів та робітників, яка здійснюється за допомогою бізнесу._

### LINKED ISSUE #36: Add introduction and business-actors characteristics
Add introduction and business-actors characteristics to `stakeholders-needs.md` with the next structure:
- Roles and proccesses characheristics:

_[В цьому розділі визначаються зовнішні фактори, що впливають на бізнес (бізнес-актори), та внутрішні фактори (робітники), дається загальна характеристика діяльності бізнес-акторів та робітників, яка здійснюється за допомогою бізнесу._

### LINKED ISSUE #47: Add use-cases: DeleteTask, BanUser, UnbanUser

### PR DESCRIPTION
**Pull request by KseniiaHorai**

### CODE REVIEW

File: docs/requirements/stakeholders-needs.md [hunk 1]
```diff
@@ -319,24 +356,37 @@
...
     </tr>
      <tr>
         <td><b>Результат:</b></td>
-        <td></td>
+        <td>Видалене завдання</td>
     </tr>
      <tr>
         <td><b>Виключні ситуації:</b></td>
         <td>
+         - RejectedAcess_EXC - користувач не має відповідного дозволу для видалення цієї задачі </br>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]:
```suggestion
         - DeleteTask_RejectedAcess_EXC - користувач не має дозволу для видалення цієї задачі </br>
```

File: docs/requirements/stakeholders-needs.md [hunk 2]
```diff
@@ -319,24 +356,37 @@
...
+         - DeleteTask_CancelButton_EXC - користувач натиснув кнопку "Відміна"
         </td>
     </tr>
     <tr>
         <td><b>Основний сценарій:</b></td>
         <td>
+          <ol>
+            <li>Користувач обирає проєкт</li>
+            <li>Користувач обирає задачу і натискає на кнопку "Видалити задачу"</li>
+            <li>Система відображає діалогове вікно для підтвердження операції видалення(можливий DeleteTask_CancelButton_EXC)</li>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]:
```suggestion
            <li>Система відображає діалогове вікно для підтвердження операції видалення (можлива DeleteTask_CancelButton_EXC)</li>
```

File: docs/requirements/stakeholders-needs.md [hunk 3]
```diff
@@ -319,24 +356,37 @@
...
+            <li>Користувач натискає кнопку "Так"</li>
+            <li>Система перевіряє права користувача (можливий RejectedAcess_EXC)</li>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]:
```suggestion
            <li>Система перевіряє права користувача (можлива DeleteTask_RejectedAcess_EXC)</li>
```

File: docs/requirements/stakeholders-needs.md [hunk 4]
```diff
@@ -824,24 +874,40 @@
...
         <td><b>Учасники:</b></td>
-        <td></td>
+        <td>Адміністратор, система</td>
     </tr>
      <tr>
         <td><b>Передумови:</b></td>
-        <td></td>
+        <td>
+         -Користувач багаторазово неправильно вводить пароль<br/>
+         -Виявлення підозрілої активності в обліковому записі користувача<br/>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]:
```suggestion
         - Адміністратор виявив підозрілу активність користувача<br/>
```

File: docs/requirements/stakeholders-needs.md [hunk 5]
```diff
@@ -824,24 +874,40 @@
...
-        <td></td>
+        <td>Заблокований користувач</td>
     </tr>
      <tr>
         <td><b>Виключні ситуації:</b></td>
         <td>
+         - BanUser_NoMatchingUser_EXC - введені дані не відповідають жодному користувачеві</br>
+         - BanUser_UserHasBeenRemoved_EXC - користувача видалено з системи</br>
+         - BanUser_UserHasBeenBanned_EXC - користувача вже заблоковано</br>
+         - BanUser_CancelButton_EXC -адміністратор натиснув кнопку "Відміна"
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]:
```suggestion
         - BanUser_CancelButton_EXC - адміністратор натиснув кнопку "Відміна"
```

File: docs/requirements/stakeholders-needs.md [hunk 6]
```diff
@@ -824,24 +874,40 @@
...
         </td>
     </tr>
     <tr>
         <td><b>Основний сценарій:</b></td>
         <td>
+          1. Адміністратор фіксує підозрілу активність користувача<br/>
+          2. Адміністратор аналізує ситуацію<br/>
+          3. Адміністратор вирішує блокувати користувача<br/>
+          4. Адміністратор заповнює спеціальну форму для блокування, вказуючи причину та термін дії блокування<br/>
+          5. Адміністратор натискає кнопку "Підтвердити" (можливе BanUser_CancelButton_EXC)<br/>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]:
```suggestion
          5. Адміністратор натискає кнопку "Підтвердити" (можлива BanUser_CancelButton_EXC)<br/>
```

File: docs/requirements/stakeholders-needs.md [hunk 7]
```diff
@@ -857,24 +923,34 @@
...
+         - UnbanUser_UserHasBeenUnbanned_EXC - користувача вже розблоковано</br>
+         - UnbanUser_CancelButton_EXC - користувач натиснув кнопку "Відміна"
         </td>
     </tr>
     <tr>
         <td><b>Основний сценарій:</b></td>
         <td>
+          1. Адміністратор фіксує потрібного користувача<br/>
+          2. Адміністратор натискає на "Розблокувати користувача" <br/>
+          3. Адміністратор натискає кнопку "Підтвердити" (можливий UnbanUser_CancelButton_EXC)<br/>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]:
```suggestion
          3. Адміністратор натискає кнопку "Підтвердити" (можлива UnbanUser_CancelButton_EXC)<br/>
```

File: docs/requirements/stakeholders-needs.md [hunk 8]
```diff
@@ -824,24 +874,40 @@
...
+         - BanUser_UserHasBeenBanned_EXC - користувача вже заблоковано</br>
+         - BanUser_CancelButton_EXC -адміністратор натиснув кнопку "Відміна"
         </td>
     </tr>
     <tr>
         <td><b>Основний сценарій:</b></td>
         <td>
+          1. Адміністратор фіксує підозрілу активність користувача<br/>
+          2. Адміністратор аналізує ситуацію<br/>
+          3. Адміністратор вирішує блокувати користувача<br/>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]: You can unite this steps

**Review by kol-oss — CHANGES_REQUESTED** [review]

**Review by kol-oss — APPROVED** [review]

File: docs/requirements/stakeholders-needs.md [hunk 9]
```diff
@@ -824,24 +874,40 @@
...
+         - BanUser_UserHasBeenBanned_EXC - користувача вже заблоковано</br>
+         - BanUser_CancelButton_EXC - адміністратор натиснув кнопку "Відміна"
         </td>
     </tr>
     <tr>
         <td><b>Основний сценарій:</b></td>
         <td>
+          1. Адміністратор фіксує підозрілу активність користувача<br/>
+          2. Адміністратор аналізує ситуацію<br/>
+          3. Адміністратор вирішує блокувати користувача<br/>
```
Comment (mari4kaa) [inline docs/requirements/stakeholders-needs.md]: Delete this step

File: docs/requirements/stakeholders-needs.md [hunk 10]
```diff
@@ -857,24 +923,34 @@
...
+         - UnbanUser_UserHasBeenRemoved_EXC - користувача видалено з системи</br>
+         - UnbanUser_UserHasBeenUnbanned_EXC - користувача вже розблоковано</br>
+         - UnbanUser_CancelButton_EXC - користувач натиснув кнопку "Відміна"
         </td>
     </tr>
     <tr>
         <td><b>Основний сценарій:</b></td>
         <td>
+          1. Адміністратор фіксує потрібного користувача<br/>
+          2. Адміністратор натискає на "Розблокувати користувача" <br/>
```
Comment (mari4kaa) [inline docs/requirements/stakeholders-needs.md]:
```suggestion
          2. Адміністратор натискає на кнопку "Розблокувати користувача" <br/>
```

File: docs/requirements/stakeholders-needs.md [hunk 11]
```diff
@@ -824,24 +874,38 @@
...
+         - BanUser_NoMatchingUser_EXC - введені дані не відповідають жодному користувачеві</br>
+         - BanUser_UserHasBeenRemoved_EXC - користувача видалено з системи</br>
+         - BanUser_UserHasBeenBanned_EXC - користувача вже заблоковано</br>
+         - BanUser_CancelButton_EXC - адміністратор натиснув кнопку "Відміна"
         </td>
     </tr>
     <tr>
         <td><b>Основний сценарій:</b></td>
         <td>
+          1. Адміністратор фіксує підозрілу активність користувача і блокує його<br/>
```
Comment (kol-oss) [inline docs/requirements/stakeholders-needs.md]:
```suggestion
          1. Адміністратор фіксує підозрілу активність користувача<br/>
```

**Review by kol-oss — CHANGES_REQUESTED** [review]

**Review by kol-oss — APPROVED** [review]

**Pull request merged**